tests/fingerprints/weiner_index.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from rdkit.Chem import GetDistanceMatrix
+from scipy.sparse import csr_array
+
+from skfp.fingerprints import WeinerIndexFingerprint
+
+
```

Comment:
This requires more tests with known values, at least:
1. Examples from the original Wiener's paper
2. Single atom molecule
3. Linear molecule, e.g. butane
4. Ring-like molecule, e.g. benzene